src/Fantomas/CodePrinter.fs

```diff
@@ -4005,67 +4003,60 @@ and genPrefixTypes astContext node ctx =
             ctx
 
 and genTypeList astContext node =
-    match node with
-    | [] -> sepNone
-    | (t, [ ArgInfo (ats, so, isOpt) ]) :: ts ->
-        let gt =
+    let gt (t, args: SynArgInfo list) =
+        match t, args with
+        | TTuple ts', _ ->
+            // The '/' separator shouldn't appear here
+            let hasBracket = not node.IsEmpty
```

Comment:
I don't understand what you mean by "the '/' separator". Could you explain that bit please?

Reply:
To be honest, I kind of found that comment when I started.
https://github.com/fsprojects/fantomas/blob/924f63899a692c58d9bf4eb014d071f10d7e5e96/src/Fantomas/CodePrinter.fs#L4031

I can't really say what it does either. Perhaps I could remove the comment.